.github/workflows/ci.yml

```
@@ -0,0 +1,35 @@
+          uv pip install --system .
+          uv pip install --system pytest pytest-cov
+      
+      - name: Run tests with coverage
+        run: pytest --cov --cov-branch --cov-report=xml
+      
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          slug: Abstract-Data/campaignfinance-2023
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)